test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -51,6 +55,80 @@ const loopbackSuccesskMock = RequestMock()
     'access-control-allow-methods': 'POST, OPTIONS'
   });
 
+const loopbackPollingLogger = RequestLogger(/introspect|probe|challenge|poll/, { logRequestBody: true, stringifyRequestBody: true });
+const loopbackRedundantPollingMock = RequestMock()
+  .onRequestTo(/\/idp\/idx\/introspect/)
+  .respond(identifyWithUserVerificationLoopback)
+  .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
+  .respond((req, res) => {
+    res.headers['content-type'] = 'application/json';
+    if (!firstPollCalledForRedundantPolling) {
+      firstPollCalledForRedundantPolling = true;
+      // specifically make the first poll call takes more time to complete
+      // so that we can verify if there will be a redundant poll call after it
+      return new Promise((resolve) => setTimeout(function() {
+        res.statusCode = '200';
+        res.setBody(identify);
+        resolve(res);
+      }, 8000));
+    } else {
+      res.statusCode = '400';
+      res.setBody(badRequestError);
+    }
+  })
+  .onRequestTo(/2000|6511\/probe/)
+  .respond(null, 500, {
+    'access-control-allow-origin': '*',
+    'access-control-allow-headers': 'X-Okta-Xsrftoken, Content-Type'
+  })
+  .onRequestTo(/6512\/probe/)
+  .respond(null, 200, {
+    'access-control-allow-origin': '*',
+    'access-control-allow-headers': 'X-Okta-Xsrftoken, Content-Type'
+  })
+  .onRequestTo(/6512\/challenge/)
+  .respond(null, 200, {
+    'access-control-allow-origin': '*',
+    'access-control-allow-headers': 'Origin, X-Requested-With, Content-Type, Accept, X-Okta-Xsrftoken',
+    'access-control-allow-methods': 'POST, OPTIONS'
+  });
+
+const loopbackEnhancedPollingMock = RequestMock()
+  .onRequestTo(/\/idp\/idx\/introspect/)
+  .respond(identifyWithUserVerificationLoopbackWithEnhancedPolling)
+  .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
+  .respond((req, res) => {
+    res.headers['content-type'] = 'application/json';
+    if (!firstPollCalledForEnhancedPolling) {
+      firstPollCalledForEnhancedPolling = true;
+      // with enhanced polling, this interval can be shorter for testing
+      return new Promise((resolve) => setTimeout(function() {
+        res.statusCode = '200';
+        res.setBody(identify);
+        resolve(res);
+      }, 1000));
```

Comment:
Same as above, I would use the await here if possible.

Reply:
sure i will try it out.